src/bot_methods.js

```diff
@@ -1,314 +1,498 @@
+        console.log("input: " + input);
+        if (input == "stop") {
+          message.channel.send("Exiting logs!");
+          return;
+        } else {
+          message.channel.send("**" + input + "**: " + problemWords[input]);
+        }
+      })
+      .catch(collected => {
+        return;
+      });
+  }
+}
+
+/**
+ * Handles the "start" command
+ * starts automatic sending wotd messages and updates word status in the file
+ * sends a new word every 24 hours (as specified in MSEC_PER_DAY constant)
+ * only an admin or moderator can use this command.
+ * @param {Message} message discord message object
+ */
+function handleCommandStart(message) {
+  if (checkIfAdmin(message)) {
+    console.log("Bot started!");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot started! A new word will be posted every 24h from now."
+    );
+
+    sendWordOfTheDay(message, true);
+
+    interval = bot.setInterval(sendWordOfTheDay, MSEC_PER_DAY, message, true);
+  } else {
+    console.log("Bot did not start, user was not a moderator or admin");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot did not start, user must be a moderator or admin to use this command."
+    );
+  }
+}
+
+/**
+ * Handles the "stop" command
+ * stops automatic sending wotd messages
+ * only an admin or moderator can use this command.
+ * @param {Message} message
+ */
+function handleCommandStop(message) {
+  if (checkIfAdmin(message)) {
+    console.log("Bot stopped!");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot stopped! New words will not be posted automatically."
+    );
+
+    bot.clearInterval(interval);
+  } else {
+    console.log("Bot did not stop, user was not a moderator or admin");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot did not stop, user must be a moderator or admin to use this command."
+    );
+  }
+}
+
+/**
+ * Handles the "help" command
+ * displays information about the bot commands in chat
+ * @param {Message} message
+ */
+function handleCommandHelp(message) {
+  console.log("Bot helped!");
+
+  let help =
+    "Commands for the word of the day bot:\n" +
+    "```" +
+    "$word x  - Searches word x in the dictionary and posts it if available\n" +
+    "$start - Starts automatic sending the word of the day - once a day, from current time (requires moderator permissions)\n" +
+    "$stop  - Stops automatic sending the word of the day (requires moderator permissions)\n" +
+    "$goodbot - Shows your appreciation for the bot\n" +
+    "$goodboy - Woof!\n" +
+    "```";
+
+  message.channel.send(help);
+}
+
+function handleChallengeMode(message) {
+  let newWord = challengeMode(message);
+  console.log(newWord[0]);
+  console.log(newWord[1]);
+  const filter = response => response.content.toLowerCase() == newWord[1].toLowerCase() || response.content.toLowerCase() == 'stop' || response.content.toLowerCase() == 'next';

```

Comment:
you use `stop` and `next` multiple times in this functions. put them in const variables at the beginning of the function (e.g. const STOP = 'stop')

Reply:
but why? replacing `'stop'` with STOP just seems tedious and unnecessary

Reply:
When using values multiple times in your code, you should replace it with one constant variable. 
This is a basic rule of thumb in programming (and definitely not tedious and unnecessary).
There are 2 reasons:
1.  At some point in time you will write the same string multiple times but one will have a typo. and you introduce a bug
2. if you want to refactor the string value to something else (happens a lot) you might forget to replace all instances of the string (or will replace it in places you didn't want to), and you introduce a bug

Reply:
for me i've only replaced values with a variable if the value text was rather long, or if it was repeated several times - however for some reason i didn't consider that with strings (especially short ones)

Reply:
but regardless - duly noted